packages/perseus/src/components/tex-buttons.tsx

```diff
@@ -27,12 +34,14 @@ const basic: ReadonlyArray<ButtonSet> = [
             +
         </span>,
         "PLUS",
+        i18n._("Plus"),
```

Comment:
So it turned out that none of Perseus' MathInput buttons had labels. I'm sure this could be cleaner, but ideally we'll be removing a lot of this in favor of the Keypad soon so I didn't think it was worth thinking too much about.